lib/uploader.ts

```diff
@@ -41,7 +39,12 @@ export class Uploader {
 
 	// Global upload queue
 	private _uploadQueue: Array<Upload> = []
-	private _jobQueue: PQueue = new PQueue({ concurrency: MAX_CONCURRENCY })
+	private _jobQueue: PQueue = new PQueue({
+		// Maximum number of concurrent uploads
+		// @ts-expect-error TS2339
```

Comment:
Why that? Also the string behind `@ts-expect-error` should explain why you expect that error :)

Reply:
tsc says property `files` does not exist on `Object`. Should I put this explanation into the comment?

Reply:
Makes sense this is a bug in the capability types, but in this case it is cleaner to just cast the type.